cosmic-settings/src/pages/power/mod.rs

```diff
@@ -86,14 +120,86 @@ fn battery_info() -> Section<crate::pages::Message> {
                 ""
             });
 
-            column::with_capacity(2)
+            widget::column::with_capacity(2)
                 .spacing(8)
                 .push(text::heading(&section.title))
                 .push(row!(battery_icon, battery_percent, battery_time).spacing(8))
                 .into()
         })
 }
 
+fn connected_devices() -> Section<crate::pages::Message> {
+    let descriptions = Slab::new();
+
+    Section::default()
+        .title(fl!("connected-devices"))
+        .descriptions(descriptions)
+        .show_while::<Page>(|page| !page.connected_devices.is_empty())
+        .view::<Page>(move |_binder, page, section| {
+            let devices: Vec<cosmic::Element<'_, _>> = page
+                .connected_devices
+                .iter()
+                .map(|connected_device| {
+                    let battery_icon =
+                        widget::icon::from_name(connected_device.battery.icon_name.clone());
+                    let battery_percent =
+                        widget::text(format!("{}%", connected_device.battery.percent));
+
+                    // FIXME is that missing from the design?
+                    // let battery_time =
+                    //     widget::text(if connected_device.battery.remaining_duration > TimeDelta::zero() {
+                    //         &connected_device.battery.remaining_time
+                    //     } else {
+                    //         ""
+                    //     });
```

Comment:
@maria-komarova I believe you looked at the original design. Is this on purpose or an oversight?

Reply:
Also, the order should possibly be **battery icon - percentage**, instead of **percentage - battery icon**, to match the system battery info above, and possibly prevent the battery icon moving when the percentage changes.

Reply:
Yeah, I thought so too. Currently I have followed the design 1:1, but happy to change if there is an issue with the design